src/plugins/ffe/templates/ffe_tournament_action_items.html

```diff
@@ -0,0 +1,38 @@
+
+<li class="dropdown-submenu">
+    <a class="dropdown-item d-flex justify-content-between" href="#">
```

Comment:
The ``href="#"`` updates the URL, can we remove it?
```suggestion
    <a class="dropdown-item d-flex justify-content-between">
```